packages/integration-sdk-core/src/errors.ts

```diff
@@ -83,6 +83,9 @@ export class IntegrationError extends Error {
     this.code = options.code;
     this.fatal = options.fatal;
     this._cause = options.cause;
+    if (options.cause?.stack) {
+      this.stack = this.stack + '\n' + options.cause.stack;
```

Comment:
Consider this package (created by Matt Lavin at LifeOmic): https://github.com/mdlavin/nested-error-stacks

Reply:
I don't really feel that package adds a lot here, so I'm going to leave it out to keep a leaner dependency tree. However, I did lift `Caused by: ` which I liked from that package.